server/index.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const getImages = require('./model.js');
+const app = express();
+const port = 3003;
+
+// app.use(express.static('public'));
+
+app.get(`/api/images/:id`, function(req, res){
+  var params = [];
+  params[0] = req.params.id;

```

Comment:
You could replace lines 9 and 10 with: `const params = [req.params.id]`

Reply:
This is a good point.